kie-drools-wb/kie-drools-wb-distribution-wars/src/main/productized/webapp/logout.jsp

```diff
@@ -5,10 +5,13 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://jakarta.apache.org/taglibs/i18n-1.0" prefix="i18n" %>
 <%
-  request.logout();
-  javax.servlet.http.HttpSession httpSession = request.getSession(false);
-  if (httpSession != null) {
-    httpSession.invalidate();
+  try {
+    request.logout();
+    javax.servlet.http.HttpSession httpSession = request.getSession(false);
+    if (httpSession != null) {
+      httpSession.invalidate();
+    }
+  } catch ( Exception e ) {
```

Comment:
Yes, self-documented code is the best approach. In that case - why not to catch java.lang.SecurityException ? WARN log could be a second part of the solution. I think we should avoid swallowing general exception without knowing what actually happened.